internal/controllers/session.go

```diff
@@ -44,16 +44,23 @@ func (c *SessionController) List(gctx *gin.Context) {
 		return
 	}
 
+	qp, err := models.NewQuestionPoolFromFile(Settings.QuestionPoolFile)
+	if handleError(gctx.Writer, err, http.StatusInternalServerError) {
+		return
+	}
+
 	viewData := struct {
 		QRCodePNG  string
 		NewQuizURL string
 		Completed  []models.Session
 		InProgress []models.Session
+		Prices     models.PricesList
```

Comment:
```suggestion
		Prizes     models.PrizesList
```

and in the rest of this PR. Also maybe `PrizeList` instead of `PrizesList` , I think it's more common term.